docs/source/app-dev/bindings-ts/index.rst

```diff
@@ -1,13 +1,14 @@
 .. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-TypeScript bindings
-###################
+DAML/JavaScript Client Libraries
```

Comment:
I’m not sure adding `DAML` in the name really helps. This is hosted on `docs.daml.com`. Anything that isn’t about DAML wouldn’t make sense here anyway.